sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -1085,47 +1085,42 @@ case class ShowCreateTableCommand(table: TableIdentifier)
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
     val catalog = sparkSession.sessionState.catalog
-    if (catalog.isTemporaryTable(table)) {
```

Comment:
The change is only removing this `if` block, but diff is bad.

Reply:
shall we keep the `if`? it doesn't hurt.